### ISSUE TITLE
shortcuts in input.conf

### ISSUE THREAD

**Issue by Joseph65** [kanishka-linux/kawaii-player#11]
Shortcuts in input.conf in the directory ./config/kawaii-player/src/input.conf don't work
i am adding my own shortcuts to that file.

**Comment by kanishka-linux** [comment]
input.conf was experimental in nature and did not work out quite well in overall setup and hence currently it has been disabled. I will try to address the issue in the next release.

**Comment by kanishka-linux** [comment]
I've added some quick fixes. First you'll need to install from latest source, then add 'MPV_INPUT_CONF=True' in the file '.config/kawaii-player/other_options.txt'. Once you'll add this line, the player will follow input.conf file for keyboard shortcuts. If some shortcuts are not written in './config/kawaii-player/src/input.conf' then it will follow global mpv shortcuts. This feature is working on latest mpv 0.26.0 . There were some problems in older versions of mpv in implementing this feature, therefore I had disabled it in the past. Now the feature looks stable, but still needs some more testing.
